### PR TITLE
MAUI 843540 - Added UG content for image editor Save and browse picker

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have added UG content for image browse and save picker feature.

### CODE REVIEW

File: MAUI/ImageEditor/events.md [hunk 1]
```diff
@@ -63,3 +63,27 @@ N> This is common for Shape and Text annotations.
 {% endhighlight %}
 
 {% endtabs %}
+
+## Browse image event
+
+This `BrowseImage` event will be triggered while browsing the image source.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/events.md:69]: This `BrowseImage` event occurs while browsing the image source. 
`Cancel`: We can restrict the default image browse picker opening by setting the `Cancel` argument `true`.

File: MAUI/ImageEditor/save.md [hunk 2]
```diff
@@ -215,4 +215,28 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+### Save picker opening event
+
+The `SavePickerOpening` event occurs while the save picker is open.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/save.md:220]: The `SavePickerOpening` event occurs while the save picker is opening on the save icon click.
`Cancel`: We can restrict the save opening by setting the `Cancel` argument to `true`. If save picker is disabled, the image will be saved in the default location.

Please refer [here](https://help.syncfusion.com/maui/imageeditor/save?cs-save-lang=1&cs-lang=csharp#save-method) to learn more about the default save location.

File: MAUI/ImageEditor/events.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/events.md]: Updated.

File: MAUI/ImageEditor/save.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/save.md]: Updated.

File: MAUI/ImageEditor/events.md [hunk 3]
```diff
@@ -63,3 +63,27 @@ N> This is common for Shape and Text annotations.
 {% endhighlight %}
 
 {% endtabs %}
+
+## Browse image event
+
+This `BrowseImage` event occurs while browsing the image source.
+Cancel: We can restrict the default image browse picker opening by setting the Cancel argument true.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/events.md:70]: `Cancel`: We can restrict the default image browse picker opening by setting the `Cancel` argument `true`.

File: MAUI/ImageEditor/save.md [hunk 4]
```diff
@@ -215,4 +215,30 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+### Save picker opening event
+
+The `SavePickerOpening` event occurs while the save picker is opening on the save icon click.
+Cancel: We can restrict the save opening by setting the Cancel argument to true. If save picker is disabled, the image will be saved in the default location.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/save.md:221]: The `SavePickerOpening` event occurs while the save picker is opening on the save icon click.
`Cancel`: We can restrict the save picker opening by setting the `Cancel` argument to `true`. If the save picker is disabled, the image will be saved in the default location.

Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/save.md]: Updated.

File: MAUI/ImageEditor/events.md [hunk 5]
```diff
@@ -63,3 +63,27 @@ N> This is common for Shape and Text annotations.
 {% endhighlight %}
 
 {% endtabs %}
+
+## Browse image event
+
+This `BrowseImage` event occurs while browsing the image source.
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/events.md:69]: This `BrowseImage` event occurs while browsing the image source when clicking on browse icon in toolbar.

File: MAUI/ImageEditor/save.md [hunk 6]
```diff
@@ -215,4 +215,30 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+### Save picker opening event
+
+The `SavePickerOpening` event occurs while the save picker is opening on the save icon click.
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/save.md:220]: save icon clicked in toolbar.

File: MAUI/ImageEditor/events.md (hunk 5 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/events.md]: Updated.

File: MAUI/ImageEditor/save.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/save.md]: Updated.

File: MAUI/ImageEditor/events.md [hunk 7]
```diff
@@ -63,3 +63,27 @@ N> This is common for Shape and Text annotations.
 {% endhighlight %}
 
 {% endtabs %}
+
+## Browse image event
+
+This `BrowseImage` event occurs while browsing the image source when clicking on browse icon in toolbar.
```
Comment (Chrispineagunja) [inline MAUI/ImageEditor/events.md:69]: The `BrowseImage` event occurs when you click the browse icon in the toolbar while browsing the image source.

File: MAUI/ImageEditor/events.md [hunk 8]
```diff
@@ -63,3 +63,27 @@ N> This is common for Shape and Text annotations.
...
+`Cancel`: We can restrict the default image browse picker opening by setting the `Cancel` argument `true`.
```
Comment (Chrispineagunja) [inline MAUI/ImageEditor/events.md:70]: `Cancel`: Restrict the default image browse picker opening by setting the `Cancel` argument `true.`

File: MAUI/ImageEditor/save.md [hunk 9]
```diff
@@ -215,4 +215,30 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+### Save picker opening event
+
+The `SavePickerOpening` event occurs while the save picker is opening on the save icon clicked in toolbar.
```
Comment (Chrispineagunja) [inline MAUI/ImageEditor/save.md:220]: The `SavePickerOpening` event occurs while the save picker opens on the save icon clicked in the toolbar.

File: MAUI/ImageEditor/save.md [hunk 10]
```diff
@@ -215,4 +215,30 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
...
+`Cancel`: We can restrict the save picker opening by setting the `Cancel` argument to `true`. If the save picker is disabled, the image will be saved in the default location.
```
Comment (Chrispineagunja) [inline MAUI/ImageEditor/save.md:221]: `Cancel`: Restrict the save picker opening by setting the `Cancel` argument to `true.` If the save picker is disabled, the image will be saved in the default location.

File: MAUI/ImageEditor/save.md [hunk 11]
```diff
@@ -215,4 +215,30 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
...
+
+Please refer [`here`](https://help.syncfusion.com/maui/imageeditor/save?cs-save-lang=1&cs-lang=csharp#save-method) to learn more about the default save location.
```
Comment (Chrispineagunja) [inline MAUI/ImageEditor/save.md:223]: Please refer to [`here`](https://help.syncfusion.com/maui/imageeditor/save?cs-save-lang=1&cs-lang=csharp#save-method) to learn more about the default save location.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

File: MAUI/ImageEditor/events.md (hunk 7 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/events.md]: updated.

File: MAUI/ImageEditor/events.md (hunk 8 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/events.md]: updated.

File: MAUI/ImageEditor/save.md (hunk 9 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/save.md]: updated.

File: MAUI/ImageEditor/save.md (hunk 10 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/save.md]: updated.

File: MAUI/ImageEditor/save.md (hunk 11 above)
Comment (TamilarasanGunasekaran) [reply MAUI/ImageEditor/save.md]: updated.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**